docs/topics/Gateway.md

```diff
@@ -38,38 +38,37 @@ Returns an object with the same information as [Get Gateway](#DOCS_GATEWAY/get-g
 The Discord Gateway has a versioning system which is separate from the core APIs. The following table specifies all versions of the Gateway API that have been officially supported, and whether or not they are out of service (e.g. unsupported and potentially disfunctional). The documentation herein is only for the latest version in the following table, unless otherwise specified.
 
 | Version | Out of Service |
-|------------|----------------|
+|---------|----------------|
 | 5 | no |
 | 4 | no |
 
-
 ## Gateway OP Codes/Payloads
 
 ###### Gateway Payload Structure
 
 | Field | Type | Description | Present |
-|-------|------|-------------|-------------|
+|-------|------|-------------|---------|
 | op | integer | opcode for the payload | Always |
-| d | mixed (object, integer) | event data | Always |
+| d | mixed (object, integer, bool, null) | event data | Usually |
 | s | integer | sequence number, used for resuming sessions and heartbeats | Only for OP 0 |
 | t | string | the event name for this payload | Only for OP 0 |
 
 ###### Gateway OP Codes
 
-| Code | Name | Description |
-|--------|----------|-----------------|
-| 0 | Dispatch | dispatches an event |
-| 1 | Heartbeat | used for ping checking |
-| 2 | Identify | used for client handshake |
-| 3 | Status Update | used to update the client status |
-| 4 | Voice State Update | used to join/move/leave voice channels |
-| 5 | Voice Server Ping | used for voice ping checking |
-| 6 | Resume | used to resume a closed connection |
-| 7 | Reconnect | used to tell clients to reconnect to the gateway |
-| 8 | Request Guild Members | used to request guild members |
-| 9 | Invalid Session | used to notify client they have an invalid session id |
-| 10 | Hello | sent immediately after connecting, contains heartbeat and server debug information |
-| 11 | Heartbeat ACK | sent immediately following a client heartbeat that was received |
+| Code | Name | d | Description |
```

Comment:
The op codes list should just be a list of codes, the responses should be documented within their individual sections. The datatype being listed here serves really no purpose